### PR TITLE
[zh-cn]: create the translation of `api.HTMLMediaElement.setSinkId()` method

### PR DESCRIPTION
**Pull request by tianyeeT**
https://github.com/mdn/translated-content/issues/20744#issuecomment-2163722099

### CODE REVIEW

File: files/zh-cn/web/api/htmlmediaelement/setsinkid/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: "HTMLMediaElement: setSinkId() method"
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlmediaelement/setsinkid/index.md:2]:
```suggestion
title: HTMLMediaElement：setSinkId() 方法
```

File: files/zh-cn/web/api/htmlmediaelement/setsinkid/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+访问此 API 受以下约束：
+
+- 方法必须在[安全上下文](/zh-CN/docs/Web/Security/Secure_Contexts)中调用。
+- 访问可能受 [`speaker-selection`](/zh-CN/docs/Web/HTTP/Headers/Permissions-Policy/speaker-selection) HTTP [权限策略](/zh-CN/docs/Web/HTTP/Permissions_Policy)的控制。
+- 用户需要权限才能访问非默认设备。用户通过选择与 {{domxref("MediaDevices.selectAudioOutput()")}} 显示的提示中 ID 相关联的设备来授予权限。
+
+## 示例
+
+以下示例展示了如何从 {{domxref("MediaDevices.enumerateDevices()")}} 返回的设备数组中选择一个音频输出设备，并将其设置为音频的输出目标。
+请注意，`enumerateDevices()` 的结果仅包括无需用户权限或已经授予权限的设备。
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlmediaelement/setsinkid/index.md:49]: [不要断行](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E6%8E%92%E7%89%88)

```suggestion
以下示例展示了如何从 {{domxref("MediaDevices.enumerateDevices()")}} 返回的设备数组中选择一个音频输出设备，并将其设置为音频的输出目标。请注意，`enumerateDevices()` 的结果仅包括无需用户权限或已经授予权限的设备。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
其他的两个 PR 也有类似的问题，请修改

**Comment by yin1999** [comment]
Blocked by #34088

**Review by skyclouds2001 — COMMENTED** [review]
https://github.com/mdn/content/pull/34088 has been merged

**Review by yin1999 — APPROVED** [review]

**Pull request merged**